CIP-1856/README.md

```diff
@@ -0,0 +1,149 @@
+---
+CIP: ?
+Title: Extend token metadata for translations
+Category: Metadata
+Status: Proposed
+Authors:
+    - Vito Melchionna <info@granadapool.com>
+    - Aaron Schmid <aaron@fortech.group>
+    - Carolina Isler @LaPetiteADA <lapetiteada@granadapool.com>
+Implementors: N/A
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/488
+Created: 2023-03-19
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta                   | For meta-CIPs which typically serve another category or group of categories.
+- Reward-Sharing Schemes | For CIPs discussing the reward & incentive mechanisms of the protocol.
+- Wallets                | For standardization across wallets (hardware, full-node or light).
+- Tokens                 | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata               | For proposals around metadata (on-chain or off-chain).
+- Tools                  | A broad category for ecosystem tools not falling into any other category.
+- Plutus                 | Changes or additions to Plutus
+- Ledger                 | For proposals regarding the Cardano ledger
+- Catalyst               | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+## Abstract
+This proposal defines an additional property to the metadata standard for tokens (NFTs and FTs) to support text localization.
+
+## Motivation: why is this CIP necessary?
+Current token metadata only supports a single hardcoded language (mostly English), which limits the accessibility to a certain culture. To get closer to mass adoption, we need to bring down language barriers by extending the current standard to support translations. This is especially relevant for games, metaverse solutions, and RealFi use cases of NFTs.
+
+## Specification
+This proposal follows the same specifications as [CIP-0025](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025).
+    
+The name of a culture consists of its [[ISO-639]](https://www.iso.org/standard/4767.html) language code with small letters and its [[ISO-3166]](https://www.iso.org/standard/63545.html) country/region code with capital letter separated by a dash "-". For instance, this proposal was written in "en-US": English with the US culture.
+
+This convention is compatible with most operative systems (Linux and Windows) and widely used translation software.
+    
+### Proposed structure
+
+The new JSON metadata standard will look like this:
+```
+{

```

Comment:
Probably it's best to update the cddl definition of CIP25 instead of a JSON specification like this

Reply:
https://github.com/cardano-foundation/CIPs/blob/master/CIP-0025/cddl/version_2.cddl

Reply:
Good idea, I'll fork that CIP and will add the new attributes once this proposal is approved.